### ISSUE TITLE
Unable to update to the latest version from v3.2.1.6628

### ISSUE THREAD

**Issue by chanduv** [gitextensions/gitextensions#7764]
#Error Details
```
System.Net.Sockets.SocketException (0x80004005): No connection could be made because the target machine actively refused it 127.0.0.1:8888
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
```

## Additional information
Unable to update to the latest version.

## Environment
- Git Extensions 3.2.1.6628
- Build bc07663c3c893f1d896520e0bdcfbe35cfe4b729
- Git 2.21.0.windows.1 (recommended: 2.22.0 or later)
- Microsoft Windows NT 10.0.16299.0
- .NET Framework 4.8.4075.0
- DPI 96dpi (no scaling)

**Comment by vbjay** [comment]
Looks like you have a proxy?

**Comment by gerhardol** [comment]
No information in the PR, all points to the following:
Duplicate of #7613

Fixed in #7576

Please update to 3.3.1
